diff -up seahorse-3.20.0/pgp/seahorse-pgp-backend.c.gpg14 seahorse-3.20.0/pgp/seahorse-pgp-backend.c
--- seahorse-3.20.0/pgp/seahorse-pgp-backend.c.gpg14	2017-02-24 13:13:37.219519356 -0500
+++ seahorse-3.20.0/pgp/seahorse-pgp-backend.c	2017-02-24 13:14:11.113770713 -0500
@@ -341,7 +341,7 @@ seahorse_pgp_backend_initialize (void)
 
 	g_return_if_fail (pgp_backend != NULL);
 
-	gpgme_set_engine_info (GPGME_PROTOCOL_OpenPGP, NULL, NULL);
+	gpgme_set_engine_info (GPGME_PROTOCOL_OpenPGP, GNUPG, NULL);
 }
 
 SeahorseGpgmeKeyring *
